### ISSUE TITLE
Keep zoom level when changing time step

### ISSUE THREAD

**Issue by HighIander** [openPMD/openPMD-viewer#140]
Currently, the zoom level restores to unzoomed when the time step is changed. This might be unconvenient when one wants to only see a detail and study its change over time.
Keeping the zoom level, just as all the other settings are kept would be very helpful.

Off-topic: Cool tool!!!!

**Comment by ax3l** [comment]
Sounds great! We have to remember it for both plots in the `slider` (field plot and histograms) I guess, making it a bit complex. Also we are currently lagging a bit the `mpl` figure handling, which leads in `%matplotlib notebook` node often to the case that later matplotlib calls interfere with the slider (independent issue).

I think the required basis to get this done is what we discussed briefly when I was last time at LBNL about a [MVC](https://en.wikipedia.org/wiki/Model-view-controller) (model-view-controller) abstraction of the slider, especially which members (figures, axes, etc.) need to be kept as members.

I did something similar last year with a small quasi-GUI tool where I kept the following members:
```python
        self.fig #1 & 2
        self.ax # 1 & 2
        self.cb # 1 & 2
        self.last_xlim = None # 1 & 2
        self.last_ylim = None # 1 & 2
        self.myCmaps = {} # 1 & 2
```

Did changes in a `draw_iteration(...)` call which only updated existing `axes` (or created them initially) and ended in
```python
self.fig.canvas.draw()
```
Members where updated with e.g. zoom information via
```python
    def save_zoom(self) :
        self.last_xlim = self.ax.get_xlim()
        self.last_ylim = self.ax.get_ylim()

    def reset_zoom(self) :
        self.last_xlim = None
        self.last_ylim = None
```
before starting a new draw (in case on can not react on an event).

[The following matplotlib docs](https://gist.github.com/ax3l/fc123cb94f59d440f952) helped me tremendously understanding all the figure vs axes vs axis vs formatter vs canvas vs ... and redraw fuzz.
